### DISCUSSION TITLE
Keep root dir the same as opening dir.

### DISCUSSION THREAD

**Discussion by ChrisTaylorDeveloper** [nvim-tree/nvim-tree.lua#2581] [Q&A] [answered]
How can I prevent the root directory of nvim-tree changing all the time?
I assume my present working directory is changing also.
I just want the working dir and the root dir of nvim-tree to remain un-changed i.e. the directory I opened lunarvim at.

**Comment by alex-courtis** [comment]
That can be done - nvim-tree root only changing when you explicitly change it via, say, `<C-]>`

In fact, that is the default behaviour.

Configuration setings like `:help nvim-tree.sync_root_with_cwd` are required to change this behaviour. Please post your config.

I believe that lunarvim does set some opinionated options.

**Reply by ChrisTaylorDeveloper** [reply]
Thank you.  In fact I have no bespoke config related to nvim-tree, everything is the Lunarvim default.  Perhaps my question should really be, how do I stop the cwd from changing, which is not really a nvim-tree problem.

**Reply by alex-courtis** [reply, marked as answer]
I've taken a look at lunarvim and it is configured with `sync_root_with_cwd`, `update_focused_file.update_root` and other tree root modifiers.

https://github.com/LunarVim/LunarVim/blob/fc9716a65f304400417bf79c00912cd670272c8a/lua/lvim/core/nvimtree.lua

You can reset / clear those options as per defaults: `:help nvim-tree-setup`
